plugins/woocommerce/i18n/states.php

```diff
@@ -804,43 +804,45 @@
 		'WW' => __( 'Wicklow', 'woocommerce' ),
 	),
 	'IN' => array( // Indian states.
+		'AN' => __( 'Andaman and Nicobar Islands', 'woocommerce' ),
 		'AP' => __( 'Andhra Pradesh', 'woocommerce' ),
 		'AR' => __( 'Arunachal Pradesh', 'woocommerce' ),
 		'AS' => __( 'Assam', 'woocommerce' ),
 		'BR' => __( 'Bihar', 'woocommerce' ),
+		'CG' => __( 'Chhattisgarh', 'woocommerce' ),
+		'CH' => __( 'Chandigarh', 'woocommerce' ),
 		'CT' => __( 'Chhattisgarh', 'woocommerce' ),
+		'DD' => __( 'Daman and Diu', 'woocommerce' ),
+		'DH' => __( 'Dādra and Nagar Haveli and Damān and Diu', 'woocommerce' ),
+		'DL' => __( 'Delhi', 'woocommerce' ),
```

Comment:
Please add in Dadra and Nagar Haveli
```suggestion
		'DL' => __( 'Delhi', 'woocommerce' ),
		'DN' => __( 'Dadra and Nagar Haveli', 'woocommerce' ),
```